app/models/testimonial.rb

```diff
@@ -0,0 +1,4 @@
+class Testimonial < ApplicationRecord
+  include Discard: :Model
+  has_one_attached :image
```

Comment:
add the validates rules
`validates :field, presence: true` 
for all fields
😄

Reply:
I agree with @DevChristians, add the `presence: true` validation in the model field where you have a `null: false`.